catroid/src/main/java/org/catrobat/catroid/cloudmessaging/CloudMessaging.java

```diff
@@ -54,6 +60,24 @@ public void showNotification(CloudMessage cloudMessage) {
 		}
 	}
 
+	public static boolean initialize(Context context) {
+		if (isCloudNotificationsEnabled(context)) {
+			NotificationSubscriber.subscribeToFirebaseServices();
+			return true;
```

Comment:
i personally like it when there are as least exit points as possible.
so you could save the `isCloudNotificationsEnabled(context)` return value in a variable, then either sub or unsub and then return the variable value in the end.

@robertpainsi do you agree or doesn't it matter?